code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -65,8 +65,6 @@
 	var/lights_power = 6
 	///Just stop the mech from doing anything
 	var/completely_disabled = FALSE
-	///Whether this mech is allowed to move diagonally
-	var/allow_diagonal_movement = FALSE
```

Comment:
hello balance change in a fix PR

Reply:
this isnt a balance change you idiot, the refactor added this in presumably by accident as it breaks mech strafing (and also doesnt make any sense at all)

Reply:
wow you didnt even mention the balance change in your own PR, guess this makes even more sense then.